pkgs/development/tools/misc/swig/3.x.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     sha256 = "1wyffskbkzj5zyhjnnpip80xzsjcr3p0q5486z3wdwabnysnhn8n";
   };
 
-  nativeBuildInputs = [ autoconf automake libtool bison ];
+  nativeBuildInputs = [ autoconf automake libtool bison pcre.dev ];
```

Comment:
Since it is in `nativeBuildInputs` I wonder why this is the case?

Reply:
pcre-config is missing when pcre is in buildInputs. See pcre drv.

Reply:
wow, rebuild linux 500+ :(

Reply:
We can push this to staging.